markdownlint-cli2-config-schema.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Is this necessary?

Reply:
I'm being explicit here, and this also enables validation while editing the schema in VS Code.